climate_scanner/trends_innovation_classifier/trends_innovation_classifier.py

```diff
@@ -44,6 +45,29 @@ def eval(self):
         """
         pass
 
+class Doc2Trends:
+
+    def __init__(self,input_dict=None,threshold=None):
+        self.input = input_dict
+        self.threshold = threshold
+
+    def coordinator_pipeline(self):
+        id_dict = dict()
+        id_dict["ID"] = self.input["ID"]
+        if self.input["climate_scanner_prob"] >= self.threshold:
+            text = self.input["doc"]
+            split_sentences = doc_to_sentence(text)
+            all_predictions = []
+            for dic in split_sentences:
+                sentence = data_processing(dic["text"])
+                tic = TrendsInnovationClassifier()
+                output = tic.predict(sentence)
```

Comment:
Ok, so at the moment we iterate through the sentences, and update with predictions for each sentence? We probably want to only keep the positive predictions in the eventual return object, so it's easier to work with. Two things worth thinking ahead on a little:
- Potentially passing on the predictions of previous/subsequent sentences as feature input (colearning), andwhether we want to roll the predictions up to paragraph level under some set of conditions. It's probably worth staring at a bunch of examples in the data to consider these questions. 
- Recovering the string indices.

Reply:
Yes, you are right! This is just the beginning, the final one would have more features. We can add them as we go.